### PR TITLE
Correct testing setup

### PR DESCRIPTION
**Pull request by bensteinberg**

### CODE REVIEW

**Comment by bensteinberg** [comment]
Closing to get PR to test at Circle...

**Comment by bensteinberg** [comment]
Reopening....

**Comment by bensteinberg** [comment]
OK, this is now passing at both Circle and Travis. I've squashed all my experiments into one commit. I don't love the two versions of `wr.env`. I also am not sure this is the best way to set up the database before testing. To be discussed.

File: services/docker/webrecorder/wr.env [hunk 1]
```diff
@@ -38,10 +38,10 @@ DISABLE_SSR=false
 # ============================
 ALLOW_EXTERNAL=1
```
Comment (rebeccacremona) [inline services/docker/webrecorder/wr.env]: I think the changes to this file can be reverted.

**Pull request merged**